visualizations/podium/components/Podium.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
-import { useProps } from "../context/VizPropsProvider";
-import { useNerdGraphQuery } from "../hooks/useNerdGraphQuery";
+import { useProps } from "../../../shared/context/VizPropsProvider/VizPropsProvider"; 
```

Comment:
you could rename the provider file to `index.tsx` and got rid of the second `/VizPropsProvider` from the import.

I'm nit picking so up to you

Reply:
thats a good idea